helix-core/src/test/java/org/apache/helix/TestListenerCallback.java

```diff
@@ -119,6 +119,8 @@ public void beforeClass() throws Exception {
 
   @AfterClass
   public void afterClass() throws Exception {
+    String testClassName = this.getShortClassName();
+    System.out.println("AfterClass: " + testClassName + " of TestListenerCallback called.");
```

Comment:
Will this bring lots of printout? Is this for debugging purpose?

Reply:
It does not really make test output much longer. The reason is that sometime, the error message is emitted from afterClass(), sometimes, from last testing method. It is hard to distinguish. 

With this log, it would be clear for people to examine